cedar-lean/DiffTest/Parser.lean

```diff
@@ -246,6 +261,26 @@ partial def exprToValue : Expr → ParseResult Value
 def jsonToValue (json : Lean.Json) : ParseResult Value :=
   jsonToExpr json >>= exprToValue
 
+
+inductive ValueOrExpr where
+  | value : Value → ValueOrExpr
+  | expr : Cedar.Partial.Expr  → ValueOrExpr
```

Comment:
What's the difference between this and
```
inductive ValueOrExpr where
  | value (v : Value)
  | expr (x : Cedar.Partial.Expr)
```

Reply:
I think the difference is just whether the fields are named.

Reply:
Yes that’s the only difference IIRC:  you get the specified names in the error messages and other help widgets.